packages/HilltopCrawler/src/main/kotlin/nz/govt/eop/hilltop_crawler/fetcher/TaskMappers.kt

```diff
@@ -206,7 +262,9 @@ class MeasurementDataTaskMapper(
    * be called when getting historical data as well)
    */
   override fun determineNextFetchAt(): Instant {
+
```

Comment:
This `determineNextFetchAt` is basically the same logic that was here previously, but still a question about how we could improve this (probably be able to do something from the time gap between observations).